Sources/TokamakCore/Shapes/Ellipse.swift

```diff
@@ -38,9 +38,19 @@ public struct Capsule: Shape {
     self.style = style
   }
 
-  public func path(in r: CGRect) -> Path {
-    .init(roundedRect: r,
-          cornerRadius: min(r.size.width, r.size.height) / 2,
-          style: style)
+  public func path(in rect: CGRect) -> Path {
+    let widthRadius = rect.size.width / 2
+    let heightRadius = rect.size.height / 2
+    return .init(
+      storage: .roundedRect(.init(
+        rect: rect,
+        cornerSize: .init(
+          width: widthRadius,
+          height: heightRadius
+        ),
```

Comment:
For a `Capsule`, wouldn’t both dimensions be  `min(widthRadius, heightRadius)`? Otherwise it’s just an ellipse.

```suggestion
        cornerRadius: min(rect.size.width, rect.size.height) / 2,
```